sdk/messaging/azeventhubs/doc.go

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Nit: Copyright header should be after build tags.

Reply:
Done.